Topics/Tech_Stacks/Shadcn.md

```diff
@@ -0,0 +1,144 @@
+# Using shadcn/ui
+
+## Table of Contents
+
+#### [Introduction](#introduction-1)
+
+#### [Installation](#installation-1)
+
+#### [Creating and Customizing Components](#creating-and-customizing-components-1)
+
+## Introduction
```

Comment:
Consider adding an "Advantages" or "Why use shadcn/ui?" section to introduce its benefits and why it's more useful than other frameworks, such as MUI.

Reply:
+1 !

Reply:
That's a great idea. I think it makes most sense to add a section like that within the Introduction section so I'll do that.